lib/std/fmt.zig

```diff
@@ -2164,6 +2180,9 @@ test "struct" {
     };
 
     try expectFmt("S{ .a = 456, .b = error.Unused }", "{}", .{inst});
+    // Tuples
+    try expectFmt("{ }", "{}", .{.{}});
```

Comment:
I would add another test case for single-item tuples as those are another edge case (no trailing comma)

Reply:
Good idea, added one (and force-pushed).